releasenotes/notes/schedule-pad-method-a56c952fcfdfbf08.yaml

```diff
@@ -0,0 +1,47 @@
+---
+features:
+  - |
+    Introduced a new pulse command ``Delay`` which may be inserted into a pulse
+    ``Schedule``. This command accepts a ``duration`` and may be added to any
+    ``Channel``. Other commands may not be scheduled on a channel during a delay.
+
+    The delay can be added just like any other pulse command. For example::
+
+      from qiskit import pulse
+      from qiskit.pulse.utils import pad
+
+      dc0 = pulse.DriveChannel(0)
+      delay = pulse.Delay(1)
+
+      sched = pulse.Schedule()
+      sched += delay(dc0).shift(1)
+
+      # build padded schedule by hand
+      ref_sched = delay(dc0) | sched
+
+      # pad schedule
+      padded_sched = pad(sched)
+
+      assert padded_sched == ref_sched
+
+    One may also pass addditional channels to be padded and a time to pad until,
+    for example::
+
+      from qiskit import pulse
+      from qiskit.pulse.utils import pad
+
+      dc0 = pulse.DriveChannel(0)
+      dc1 = pulse.DriveChannel(1)
+
+      delay = pulse.Delay(1)
+
+      sched = pulse.Schedule()
+      sched += delay(dc0).shift(1)
```

Comment:
can we make this a `SamplePulse`? I kept looking for a pulse, expecting the padding to go around the pulse. With everything being a delay, its harder to see that line 42 is the line that is doing the "interesting work". It would also probably be closer to real use cases, since this resulting schedule is the same as an empty schedule

Reply:
sure